packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -999,17 +991,14 @@ export default{
         [32m%s[0m
         If you think this is a bug then please create an issue at https://github.com/cloudflare/wrangler2/issues/new."
       `);
-      expect(error).toMatchInlineSnapshot(
-        `[Error: The asset path key "assets/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/folder/file.3da0d0cd12.txt" exceeds the maximum key size limit of 512. See https://developers.cloudflare.com/workers/platform/limits#kv-limits",]`
-      );
     });
   });
 
   describe("custom builds", () => {
     it("should run a custom build before publishing", async () => {
       writeWranglerToml({
         build: {
-          command: `echo "custom build" && echo "export default { fetch(){ return new Response(123) } }" > index.js`,
+          command: `node -e "console.log('custom build'); require('fs').writeFileSync('index.js', 'export default { fetch(){ return new Response(123) } }')"`,
```

Comment:
This test was specifically to also confirming that conjugated commands with `&&` ran as expected. I'm guessing you rewrote this to be x-platform compatible. Is there anything we can do to test conjugated commands, but x-platform?

Reply:
We can have an OS specific test. But the `&&` is not a valid operator in Windows command line.

Reply:
Oh well. Should be fine.

Reply:
I've added a test...